docs/csharp/language-reference/preprocessor-directives/preprocessor-pragma-warning.md

```diff
@@ -32,7 +32,7 @@ ms.assetid: 723493d5-9753-4cec-babb-54e2b8eb36b6
 // pragma_warning.cs  
 using System;  
   
-#pragma warning disable 414, CS3021  
+#pragma warning disable IDE0044, CS3021  
```

Comment:
Instead of `IDE0044` this should disable `CS0414`.  The original, which only had `414` is misleading.